eva-server/pom.xml

```diff
@@ -55,11 +55,35 @@
         <dependency>
             <groupId>org.opencb.opencga</groupId>
             <artifactId>opencga-storage-mongodb</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.mongodb</groupId>
+                    <artifactId>mongo-java-driver</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
             <groupId>uk.ac.ebi.eva</groupId>
             <artifactId>variation-commons</artifactId>
             <version>0.2-SNAPSHOT</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.springframework.boot</groupId>
+                    <artifactId>spring-boot-starter-data-jpa</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.springframework.boot</groupId>
+                    <artifactId>spring-boot-starter-data-mongodb</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.mongodb</groupId>
+                    <artifactId>mongo-java-driver</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.opencb.opencga</groupId>
+                    <artifactId>opencga-storage-mongodb</artifactId>
```

Comment:
Is there any class apart from [MongoDBObjectToVariantEntityConverter](https://github.com/EBIvariation/variation-commons/blob/develop/src/main/java/uk/ac/ebi/eva/commons/models/converters/data/MongoDBObjectToVariantEntityConverter.java) using OpenCGA code?